misc/emacs/juttle-derived-mode.el

```diff
@@ -14,7 +14,7 @@
 ;; (setq juttle-run-dir "/home/<user>/src/juttle/bin")
 
 (defvar juttle-run-dir "")
-(defvar juttle-run-prog "demo")
+(defvar juttle-run-prog "outrigger-client push --topic mydev --path")
```

Comment:
This is a prefix--the name of the program gets added afterward, and it's based on whatever buffer you're editing.